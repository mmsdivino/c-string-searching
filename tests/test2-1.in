Levenshtein Levenshtein i
e Vladimir Levenshtein 
Levenshtein 
pairwise 
