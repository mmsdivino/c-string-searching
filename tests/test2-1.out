Levenshtein einstein
Levenshtein einstein
i a
e a
Vladimir claimer
Levenshtein einstein
Levenshtein einstein
pairwise airlike
